cfme/tests/ssui/test_ssui_myservice.py

```diff
@@ -49,3 +48,15 @@ def test_myservice_crud(setup_provider, context, order_catalog_item_in_ops_ui):
         myservice = MyService(appliance, service_name)
         myservice.update({'name': '{}_edited'.format(service_name)})
         # TODO - add rest of myservice crud like delete in next phase.
+
+
+@pytest.mark.uncollectif(lambda: current_version() < '5.8')
+@pytest.mark.parametrize('context', [ViaSSUI])
+def test_vm_console(setup_provider, context, configure_websocket, order_catalog_item_in_ops_ui):
+    """Tests Myservice VM Console in SSUI."""
+    appliance = get_or_create_current_appliance()
```

Comment:
appliance = get_or_create_current_appliance() is not needed anymore as appliance is a global fixture now .

def test_vm_console(appliance ,setup_provider, context, configure_websocket, order_catalog_item_in_ops_ui):
this will do

Reply:
shall we update test_myservice_crud as well with this ?